src/core_landice/Registry.xml

```diff
@@ -165,6 +165,42 @@
                         description="If true, then ice from small grounded islands is removed.  Specifically, this finds one- and two-cell masses of ice that are surrounded by open ocean and eliminates them by sending them to the calving flux.  Grounded such ice masses would be islands, floating would be icebergs.  The ice removed is added to the calvingThickness field."
 	                possible_values=".true. or .false."
 		/>
+		<nml_option name="config_restore_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the restore_damage function."
+		            possible_values=".true. or .false."
+                />
+		<nml_option name="config_calculate_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the calculate_damage function."
+		            possible_values=".true. or .false."
+                />
+                <nml_option name="config_damage_restore_threshold" type="real" default_value="0.0" units="unitless"
+                        description="It is a value above which the old damage number would be kept from the impact of advection (the damage would not heal). For example, if the damage at some location is 0.83, and we set config_damage_restore_threshold to 0.8, then the damage number there will not fall below 0.83. Otherwise, if the damage is 0.7, then it could be decreased to 0.6 if non-damaged ice is transported from upstream. If we set config_damage_restore_threshold to 0 (default), then it will just not play a role in pratical."
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_calving_threshold" type="real" default_value="1.0" units="unitless"
+                        description="It is a value above which the cell would be calved away."
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_stiffness_min" type="real" default_value="0.1" units="unitless"
+                        description="When coupling damage to rheology, we do not allow it to fall below this value"
```

Comment:
Whether or not there are scientific arguments for letting damage heal below the Nye value, I know that Jeremy suggests that the Nye value is the min. value (if damage healed to zero, then no new damage can form -- you need some damage to begin with to evolve it further, which is why they choose some min. value everywhere). So at least in keeping with the physics we're trying to implement (in Jeremy's papers) I think this min needs to be the Nye value. To be clear, I think for now this is only assumed to be the case on an ice shelf (not for grounded ice).

Reply:
> Does the damage have a minimum value that it cannot drop below in the damage evolution scheme? Or does the rheology coupling just use the maximum value between the actual damage and this threshold?

The coupling is using a stiffness value that equals (1-damage), but if it is < the threshold, we'll just use this threshold as stiffness. The purpose is to avoid a too small stiffness value for the highly damaged region for a reason of velocity solver convergence.

Reply:
I think this is fine to leave in as an option as long as the reason for it is clearly explained (as you describe here). If there is guidance from other work (e.g. Sanain and Frank's paper(s)) for what the min. value should be maybe that could be noted too. You could note that this is (I think) essentially a regularization because if the damage is 1 and the ice stiffness is defined as 1-damage, then obviously you have a case where the ice stiffness goes to zero, which we clearly don't want / can't have.

Reply:
I add more info for this entry

Reply:
How about replacing the first sentence with "The minimum value the stiffnessFactor can take due to evolving damage when stiffnessFactor is calculated from damage (config_damage_rheology_coupling=.true.).  Used to prevent unrealistic and/or numerically challenging stiffness values."